l10n_ch_qr_no_amount/models/res_partner_bank.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models
+
+
+class ResPartnerBank(models.Model):
+
+    _inherit = "res.partner.bank"
+
+    def _prepare_swiss_code_url_vals(
+        self, amount, currency_name, debtor_partner, reference_type, reference, comment
+    ):
+        res = super()._prepare_swiss_code_url_vals(
+            amount, currency_name, debtor_partner, reference_type, reference, comment
+        )
+        if self.env.context.get("_no_amount_qr_code", False):
+            res = self._remove_swiss_code_amount(res)
+        return res
+
+    def _remove_swiss_code_amount(self, values_list):
```

Comment:
```suggestion
    @api.model
    def _remove_swiss_code_amount(self, values_list):
```

Reply:
As it's not done in the super methods, I guess we can avoid adding it if unnecessary:
https://github.com/odoo/odoo/blob/13.0/addons/l10n_ch/models/res_bank.py#L192
https://github.com/odoo/odoo/blob/13.0/addons/l10n_ch/models/res_bank.py#L264